generators/client/templates/vue/src/main/webapp/app/core/jhi-navbar/jhi-navbar.vue.ejs

```diff
@@ -20,7 +20,6 @@
                 <b-nav-item-dropdown
                     id="entity-menu"
                     v-if="authenticated"
-                    :class="{'router-link-active': subIsActive('/entity')}"
```

Comment:
I removed this line as I don't know how to update with this change. If someone has or can find a solution, it would be really appreciated ! :smile:

Reply:
Indeed, I don't know either

Reply:
I woul say let's leave it not marked active as adding all entities to the check sounds a bit over the top.